gnosis/cowswap
# added auto-connect to latest wallet

# Summary

Fixes #1496

This PR will enable auto connecting to the last wallet used in the app on page reload. The name of the last provider used will be stored inside of localStorage.

  # To Test

1. Connect with WalletConnect (Gnosis Safe or Metamask mobile app)
2. Reload the page and you should be auto connected with WalletConnect
3. Change connection to Metamask
4. Reload the page and you should be auto connected with Metamask
